charts/lh-operator/templates/role.yaml

```diff
@@ -22,9 +22,11 @@ rules:
 - apiGroups: ["littlehorse.io"]
   resources: ["*"]
   verbs: ["*"]
+{{- if .Values.strimzi.enabled }}
```

Comment:
```suggestion
{{- if .Values.gatewayApi.enabled }}
```

Reply:
Great catch! Let me update it right now :)

Reply:
Wait a minute how did I miss that? I tested it. Thanks Mijaíl!

Reply:
Oh, I see. This is the `Role` not the `Deployment`, which is why my tests worked as expected. I didn't test the case when `strimzi.enabled=false` and `gatewayApi.enabled=true`. That one would have failed.

Thansk @MateoRojas and @mijailr